pype/plugins/global/publish/integrate_new.py

```diff
@@ -449,6 +460,14 @@ def register(self, instance):
             self.log.debug("__ destination_list: {}".format(destination_list))
             instance.data['destination_list'] = destination_list
             representations.append(representation)
+            published_representations[repre_id] = {
+                "representation": representation,
+                "anatomy_data": template_data,
+                "published_files": published_files,
+                # TODO prabably should store subset and version to instance
```

Comment:
Probably yes. Theoretically a single instance should always only be a single subset and version at the time of publishing.